### PR TITLE
Remove boilerplate

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

File: std/equal.m31 [hunk 1]
```diff
@@ -375,6 +375,16 @@ let process_eta =
     | |- ?e : ?t => deep [] e t
     end
 
+let process_reducing c l =
+  let _ = match c with
+      | |- _constant _ => ()
+      | |- _atom _ => ()
+      | |- _proj _ _ => ()
+      | _ => print "only constants and atoms may have reducing information"; failure
```
Comment (andrejbauer) [inline std/equal.m31]: Wouldn't it be better to have `failure "only constants and atoms may have reducing information"`?

Comment (SkySkimmer) [inline std/equal.m31]: For unhandled operations we only print the name of the operation. So the error message would be uninformative.

Comment (andrejbauer) [inline std/equal.m31]: Well then, we should/could better than that. If you think about how the code is written, it's better to have `failure XXX` than `print XXX ; failure`. What would be needed? Maybe a builtin operation that fails gracefully?

Comment (SkySkimmer) [inline std/equal.m31]: In Value.top_handle (https://github.com/Andromedans/andromeda/blob/master/src/nucleus/value.ml#L606) print the `vs`

Comment (andrejbauer) [inline std/equal.m31]: Ok, that'll be a good start. Eventually we'll want something better, I suspect (such as printing of locations of terms, etc.)

**Pull request merged**